core/market/src/db/dao/negotiation_events.rs

```diff
@@ -40,17 +40,28 @@ impl<'c> AsDao<'c> for NegotiationEventsDao<'c> {
 }
 
 impl<'c> NegotiationEventsDao<'c> {
-    pub async fn add_proposal_event(
+    pub async fn add_proposal_event(&self, proposal: &Proposal, role: Owner) -> DbResult<()> {
+        let event = MarketEvent::from_proposal(proposal, role);
```

Comment:
we need to create it under the transaction and with a timestamp on the rust code side, sam as we did with agreement events

Reply:
referenced in #917